### ISSUE TITLE
Need test for #59

### ISSUE THREAD

**Issue by phated** [gulpjs/glob-stream#60]
The change introduced in https://github.com/gulpjs/glob-stream/commit/6ef29b6b41ae3da054ef17d310aa55c07f3748d3 should get a regression test.

**Comment by phated** [comment]
@danivek can you help with this?

**Comment by phated** [comment]
@doowb do you have any thoughts on implementing a regression test for this with your recent changes?

**Comment by doowb** [comment]
@phated I just noticed this...

From looking at the tests, most of them check `file.base` which uses the `basePath` that's calculated by the changes that I made. All of the tests add `path.sep` to the expected directory, so I think this case should be covered. Other than checking `file.base`, I'm not sure if there's another way to target the `basePath` variable that's created inside the function.

The other option is to move most of those utility functions into a `utils.js` file that can be individually unit tested. This may provide more coverage and reduce regression bugs.
